src/cryptonote_core/blockchain.cpp

```diff
@@ -1393,7 +1393,7 @@ bool Blockchain::validate_miner_transaction(const block& b, size_t cumulative_bl
     return false;
   }
   // From hard fork 2, we allow a miner to claim less block reward than is allowed, in case a miner wants less dust
```

Comment:
The comment is out of date. Could say
```
// From hard fork 2, we allow a miner to claim less block reward than is allowed, in case a miner wants less dust; after hard fork 6 dust became irrelevant due to RingCT, and in hard fork 13 we returned to requiring exact block rewards.
```